### ISSUE TITLE
Implement due only filter in ANC register

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-chw-anc#8]
Exactly the same functionality as in family and child registers: https://github.com/OpenSRP/opensrp-client-chw/issues/41.

------------------------------------------------------------------------------------------------------
BA: No difference from this.

**Comment by cagulas** [comment]
QA 6 June 2019:

- [ ] Doesn't seem to be working yet. I registered a woman as ANC, completed a home visit for her, then I registered a new woman, and she has a due home visit. When I select the due only filter, both women continue to be displayed in the list, when it should only be the first woman with a due visit.

![Screenshot_20190607-120459](https://user-images.githubusercontent.com/3488986/59093397-84b42980-891c-11e9-9727-a024bad78b63.png)
